vexim/login.php

```diff
@@ -88,13 +73,18 @@
     header ('Location: site.php');
     die();
   }
+
   if ($row['admin'] == '1') {
     header ('Location: admin.php');
     die();
   }
 
-  # must be a user, send them to edit their own details
-  header ('Location: userchange.php');
-
+  # must be a user, send them to edit their own details, if User-Login is permitted
+  if($AllowUserLogin===1) {
+    header ('Location: userchange.php');
+    die();
+  } else {
+    header ('Location: index.php?login=failed');
```

Comment:
Perhaps we should change this to `?login=disabled` or something similar instead?